examples/with-css-modules/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "with-css-modules",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
This gets updated, because `@markprompt/react` gets updated. `with-markprompt-web` doesn’t seem to get updated, probably because it uses `*` version ranges. Should we just use `*` version ranges for all examples’ Markprompt dependencies?

Reply:
Yes, makes sense to me 👍